lib/phoenix_live_view/channel.ex

```diff
@@ -595,6 +598,19 @@ defmodule Phoenix.LiveView.Channel do
     {:stop, :shutdown, :no_session}
   end
 
+  defp check_version_match(params) do
+    server_version = Keyword.get(project(), :version, :not_found)
+    client_version = Map.get(params, "lv_version", :not_found)
+    should_display_warning = Application.get_env(:phoenix_live_view, :warn_version_mismatch, true)
+
+    if should_display_warning && server_version != client_version do
+      Logger.warn("""
+      There is a mismatch between the LiveView version on the server (#{server_version}) and the client (#{client_version}).
+      The client side can be updated to match by running `npm install --prefix assets`.
```

Comment:
This assumes it's the client side that needs updating. Could it ever be the case that you've updated the client-side and the server side falls behind?

Reply:
Regardless if the client is behind or ahead, "syncing" the version uses the same command. :)

Reply:
Oh yeah, I forgot it's a file dependency pointing to `../deps` rather than some published package :thumbsup: